Resources/views/pjax_admin.html.twig

```diff
@@ -25,3 +25,18 @@
     </header>
     {% block content %}{% endblock %}
 </div>
+
+{% block notification %}
+<div data-name="notification" data-state="{{ notification_state|default("done") }}">
```

Comment:
after thinking a bit about it, the `data-name="notification"` element should probably be a `ul` and each notification text should be in a `li` (no matter if there's one or more notifications)
But I'm also bothered by the fact there's only a state for the list of notifications, isn't it possible to get several messages with different state ? In addition, the `notification_state` variable seems a bit magic, how does it get there ?